lib/bolt/transport/ssh.rb

```diff
@@ -16,18 +16,21 @@ def initialize
         rescue LoadError
           logger.debug("Authentication method 'gssapi-with-mic' (Kerberos) is not available.")
         end
-
-        @transport_logger = Logging.logger[Net::SSH]
-        @transport_logger.level = :warn
       end
 
       def with_connection(target)
-        conn = Connection.new(target, @transport_logger)
+        conn = if target.transport_config['ssh-command']
+                 ExecConnection.new(target)
+               else
+                 transport_logger = Logging.logger[Net::SSH]
```

Comment:
Why does it need to be in the init method if we never need it? This just seemed like a more efficient and sensible place for it.

Reply:
`Logging.logger[Net::SSH]` gets a reference to a single logging object, so it seems strange for each connection to set the log level each time, but I don't have strong feelings about it.

But if we're doing it more than once rather than initializing it and passing it to each Connection, then it can 
be in the `Connection#initialize` method so this class doesn't need to know about it.

Reply:
Oh, that's a good point, I didn't consider this would initialize a logger for every target. I'll revert it to how it was.